### ISSUE TITLE
Office UI Fabric site does not work well in High contrast

### ISSUE THREAD

**Issue by jgage176** [OfficeDev/office-ui-fabric-react#10952]
Switching to high contrast (I used the black one) the site had many accessibility issues, failing MS accessibility standards. Here are some of them I have seen and captured:

1. Pictures on left navigation for iOS and Android do not have enough color contrast:
 
![image](https://user-images.githubusercontent.com/29739628/67426580-3c299f80-f58f-11e9-9455-d183db68b08e.png)

2. Active and focus buttons on navigation have background and font color as black:
 
![image](https://user-images.githubusercontent.com/29739628/67426587-4055bd00-f58f-11e9-9f97-5d845b1e30f0.png)

3. When moving around to select a component from a menu on the left the focus is invisible:
 
![image](https://user-images.githubusercontent.com/29739628/67426598-451a7100-f58f-11e9-879f-9359443b4809.png)

Notice how the link at the bottom shows what I am currently selecting but there is no indication visually what I have selected.

4. On the individual component pages they have color contrast issues as well:
Both the active and the focus have font color and background

![image](https://user-images.githubusercontent.com/29739628/67426610-4c417f00-f58f-11e9-9218-050a0ea36f27.png)

5. The button color is white and the font color is white
 
![image](https://user-images.githubusercontent.com/29739628/67426619-519ec980-f58f-11e9-915c-600698b73077.png)

**Comment by jdhuntington** [comment]
Thanks for the report. Will dig into this this week.

**Comment by ecraig12345** [comment]
@jdhuntington Getting rid of the background gradient in high contrast might help fix some of these issues, and is something we should do regardless. The change is as follows (and you should probably do the same in `example-app-base` `_base.scss`)--the important part is the `@include high-contrast`.

```diff
diff --git a/apps/fabric-website/src/styles/_base.scss b/apps/fabric-website/src/styles/_base.scss
index 7e99df4ef..eeeeed78d 100644
--- a/apps/fabric-website/src/styles/_base.scss
+++ b/apps/fabric-website/src/styles/_base.scss
@@ -6,15 +6,21 @@
   html, body {
     -webkit-font-smoothing: antialiased;
     -webkit-tap-highlight-color: transparent;
-    background: $ms-color-gray10; // Match header/footer color for browsers with momentum scroll
+    // Match header/footer color for browsers with momentum scroll
+    background: $ms-color-gray10;
+    // Used to give the nav a white background and the body a gray background
     background: linear-gradient(to right, $ms-color-white 0%, $ms-color-white 50%, $ms-color-gray10 50%, $ms-color-gray10 100%);
     height: 100%;
     margin: 0;
     padding: 0;
     width: 100%;
+
+    @include high-contrast {
+      // The gradient doesn't behave well in high contrast, so get rid of it.
+      background: WindowBackground;
+    }
   }
 
-  // Border box everywhere
   html {
     box-sizing: border-box;
   }
```

**Comment by jdhuntington** [comment]
some of these issues tracked at #10911
